Tests/CommonHelpers/CMakeLists.txt

```diff
@@ -6,7 +6,9 @@ add_library(
   Acts/Tests/CommonHelpers/DataDirectory.cpp)
 target_compile_definitions(
   ActsTestsCommonHelpers
-  PRIVATE "ACTS_TEST_DATA_DIR=\"${acts_test_data_dir}\"")
+  PRIVATE
+    "ACTS_TEST_DATA_DIR=\"${acts_test_data_dir}\""
+    BOOST_FILESYSTEM_NO_DEPRECATED)
```

Comment:
What's this?

Reply:
This ensures that we are not accidentally using deprecated features. Recommended by the [boost documentation](https://www.boost.org/doc/libs/1_69_0/libs/filesystem/doc/deprecated.html)

Reply:
ok